pages/index.tsx

```diff
@@ -34,41 +38,131 @@ export default function Home() {
+            </div>
+          </div>
+          <div className={styles.section_two}>
+            <div className={`max_width ${styles.content}`}>
+              <div className={styles.card}>
+                <img src="/assets/icons/discord.png" alt="discord icon" />
+                <h3>Discord Community</h3>
+                <p>
+                  We have a <a>discord</a> channel where you can chat and learn with other developers.
+                </p>
+              </div>
+              <div className={styles.card}>
+                <img src="/assets/icons/open_source.png" alt="open source icon" />
+                <h3>Open Source</h3>
+                <p>
+                  My videos are almost all supported by a GitHub repo, this <a>site</a> as well.
+                </p>
+              </div>
+              <div className={styles.card}>
+                <img src="/assets/icons/education.png" alt="books education icon" />
+                <h3>Education for free</h3>
+                <p>The goal is to have enough content out for free so anyone can start learning.</p>
+              </div>
+            </div>
+          </div>
+          <div className={styles.section_three}>
+            <div className={`max_width ${styles.content}`}>
+              <h2 className={styles.title}>The Developers</h2>
+              <div className={styles.testimonials}>
+                <img src="/assets/images/testimonials/mikerydstrom.png" alt="mike rydstrom" />
+                <img src="/assets/images/testimonials/aymanbarghout.png" alt="ayman barghout" />
+                <img src="/assets/images/testimonials/tadaspetra.png" alt="tadas petra" />
+                <img src="/assets/images/testimonials/mukaldadwhal.png" alt="mukal dadwhal" />
+                <img src="/assets/images/testimonials/luischodiman.png" alt="luis chodiman" />
+              </div>
+            </div>
+          </div>
+          <div className={styles.section_four}>
+            <div className={`max_width ${styles.content}`}>
+              <img className={styles.sprite} src="/assets/icons/sprite_nervous.png" alt="sprite nervous" />
+              <h3 className={styles.title}>
+                Courses coming <span>soon</span>
+              </h3>
+              <p className={styles.desc}>
+                They will be available here. Sign up now to be notified when they are released 😃
+              </p>
+              <form
+                action="https://gmail.us2.list-manage.com/subscribe/post?u=ff73d806dd2f49da87ede8337&amp;id=ed4e712aca"
+                method="post"
+                name="mc-embedded-subscribe-form"
+                target="_blank"
+                noValidate
+              >
+                <input id="email" type="email" placeholder="johndoe@email.com" name="EMAIL" required />
+                <input id="submit" type="submit" value="Notify me" name="subscribe" />
+              </form>
```

Comment:
Create a component of this as it will later on also be used in the blog post section and other places as well